platforms/web/lib/useWysiwyg.inputEventProcessor.test.tsx

```diff
@@ -48,41 +48,25 @@ describe('inputEventProcessor', () => {
         });
     });
 
-    /**
-     *
-     * This fails in a flaky way. I (AndyB) spent quite a bit of time today
-     * trying to figure out why it's failing, and look for a workaround. The
-     * situation improves if you add other interactions e.g. send a `type` user
-     * event, but it still fails from time to time.
-     *
-     * There must be something we should be waiting for or similar but I really
-     * can't figure it out. I also can't see why this test is problematic and
-     * others are not.
-     *
-     * If the test is run on its own, it tends to pass, but if others are
-     * running, it is less reliable. This leads me to suspect there is a timing
-     * issue.
-     */
-    it.skip('Should call inputEventProcessor', async () => {
+    it('Should call inputEventProcessor', async () => {
```

Comment:
I tried to make this work in what I thought was the spirit of the original test in order to remove the only use of `.skip`.

My guess is that the issues were coming from the fact we had multiple assertions inside the waitFor block (where the callback can be run multiple times).